src/app/oqt/query-form/oqt-api-query-form/attribute-completeness-attributes/attribute-completeness-attributes.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="field">
+  <h5 style="margin-top: 0.8rem">Check the topic "{{ topicName }}" on having the following attribute(s):</h5>
+  <select multiple required
+          class="ui search selection dropdown"
+          id="search-select-attribute"
+          name="{{indicatorKey}}--attributes"
+          [(ngModel)]="selectedAttributeKeys">
+    <option *ngFor="let entry of getAttributesByTopicKey(this.selectedTopicKey) | keyvalue;"
+            [value]="entry.key">{{ entry.value.name }}
+    </option>
+  </select>
+</div>
+
+<!-- modal window to display detailed attribute information-->
+<div id="attribute-details" class="ui modal">
+  <i class="close icon"></i>
+  <div class="header">
+    <div id="title">Title will be set automatically</div>
+  </div>
+  <div class="content">
+    <div class="ui three column divided grid">
+<!--      <div class="column">-->
+<!--        <h4 class="ui header">Description:</h4>-->
+<!--        <div id="description">Description text will be set automatically</div>-->
+<!--      </div>-->
```

Comment:
Comment needed for future work?

Reply:
Yes, it's a future implementation. The information is just not yet available in the backend

Reply:
Yes. We will have this in the future once the information is available in the backend